### ISSUE TITLE
Design Driver/Operator Station

### ISSUE THREAD

**Issue by ArvickC** [FRC-1294/frc2024#120]

**Comment by SanchitArora44** [comment]
Sanchit's Version:
<img width="889" alt="image" src="https://github.com/FRC-1294/frc2024/assets/112989796/cb930903-1ed3-49e1-9313-5189e8e5d6b0">
* 38" long driver station
* handles
* 3/4" thick wooden top, 1/8" allum bottom and sides
* plenty of storage area for wires and stuff
* Indented plate for secure laptop mounting
* spots for two 3D printed Xbox controller holders
* spots for 16 buttons for operator use
* Driver joysticks, if used, can be placed on the side, off the driver station.
* Cheap, easy and quick to mfg design

**Comment by FBurkelwsd** [comment]
this seems like driver station should be in a different project. not under pit design. unless we want to have a place to attach this to inbetween matches

**Comment by SanchitArora44** [comment]
I agree. I think the plan was to be able to store the driver station on the robot cart as many other team's do.

**Comment by ArvickC** [comment]
Another design:
https://cad.onshape.com/documents/e6e526814530b6287f493f92/w/28f3ebb8da4ff0aefc9356fc
